Sources/Apodini/Environment/Environment.swift

```diff
@@ -54,6 +62,11 @@ public struct Environment<K: KeyChain, Value>: Property {
         self.keyPath = keyPath
     }
     
+    /// Initializer of `Environment` specifically for `Application` for less verbose syntax.
+    public init(_ keyPath: KeyPath<K, Value>) where K == Application {
```

Comment:
nit: For a future version of this I think we could make Application work identically to Environment. Because if I understand this PR correctly the key paths are different, right?